src/index.jsx

```diff
@@ -3,10 +3,13 @@ import React from 'react';
 import { Provider } from 'react-redux';
 import App from './App';
 import store from './Mainstore/Store';
+import { ToastContainer } from 'react-toastify';
```

Comment:
@neelxie I remember like the PO suggested we don't use toastify and instead display the status messages on the same page as @jennizalwango and @Emmanuelaaron have implemented